_site/index.html

```diff
@@ -105,61 +105,34 @@ <h2 class="section__header">Upcoming Events</h2>
         </div>
       </div>
     </section>
-    
-
-    
-    <section class="section section-past section--dark">
-
-      <div class="section__content">
-      <h2 class="section__header">WHack Winners</h2>
-      <div class="section__content" style="display: table;">
-        <div class="cardw" style="display: table-cell;">
-          <img src="/img/Whack/1st.jpg" alt="Avatar" style="width:100%">
-          <div class="containerw">
-            <h4><b>Team 17</b></br>Gunna Gunna Mamidi</h4>
-          </div>
-        </div>
-      </div>
-      </br>
-        <div class="section__content" style="display: table;">
-        <div class="cardw" style="display: table-cell;">
-            <img src="/img/Whack/2nd.jpg" alt="Avatar" style="width:100%">
-            <div class="containerw">
-              <h4><b>Team 15</b></br>First Order</h4>
-            </div>
-        </div>
-        <div class="cardw" style="display: table-cell;">
-              <img src="/img/Whack/2nd.jpg" alt="Avatar" style="width:100%">
-              <div class="containerw">
-                <h4><b>Team 5</b></br>ALPHA</h4>
-              </div>
+-->
+        
+      <section class="notes_head section section-notes">
+        <div class="button_cont" align="center">
+          <h2 class="title__heading">Reference Material</h2>
```

Comment:
Add some text under this heading in a paragraph speaking about what "reference material" actually is like the following.

> Presentations, PDFs, and other resources from our existing events